packages/ra-ui-materialui/src/field/TranslatableFieldsTabContent.tsx

```diff
@@ -30,7 +37,7 @@ export const TranslatableFieldsTabContent = (
     } = props;
     const { selectedLocale } = useTranslatableContext();
 
-    const parentSourceContext = useSourceContext();
+    const parentSourceContext = useOptionalSourceContext();
     const sourceContext = React.useMemo(
         () => ({
             getSource: (source: string) =>
```

Comment:
We shouldn't do anything special here. The sourceContext is only useful for the label.